applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bdf_scheme_turbulent.h

```diff
@@ -134,8 +131,7 @@ namespace Kratos {
           Scheme<TSparseSpace, TDenseSpace>(),
           mpTurbulenceModel(pTurbulenceModel),
           mRotationTool(DomainSize,DomainSize+1,IS_STRUCTURE,0.0) // Second argument is number of matrix rows per node: monolithic elements have velocity and pressure dofs
```

Comment:
Do we want to keep this legacy variable?

Reply:
No, but this is a change that needs to be done at once in the entire codebase and in a coordinated way with c2c (see #214).